Sources/Plasma/FeatureLib/pfGLPipeline/plGLEnumerate.cpp

```diff
@@ -261,6 +265,16 @@ void plCGLEnumerate(std::vector<hsG3DDeviceRecord>& records)
         hsG3DDeviceRecord devRec;
         devRec.SetG3DDeviceType(hsG3DDeviceSelector::kDevTypeOpenGL);
         devRec.SetDriverName("OpenGL.framework");
+        
+        plDisplayHelper* displayHelper = plDisplayHelper::CurrentDisplayHelper();
+        for (const auto& mode : displayHelper->GetSupportedDisplayModes(mainDisplay)) {
+            hsG3DDeviceMode devMode;
+            devMode.SetWidth(mode.Width);
+            devMode.SetHeight(mode.Height);
+            devMode.SetColorDepth(mode.ColorDepth);
+            devRec.GetModes().emplace_back(devMode);
```

Comment:
This is still making a copy. You'll need to `std::move(devMode)` or supply the arguments to `emplace_back()` to construct in place.

Reply:
hsG3DDeviceMode doesn't have a constructor. I'll std::move it for now.